python/rmm/rmm.py

```diff
@@ -231,3 +239,40 @@ def rmm_cupy_allocator(nbytes):
     ptr = cupy.cuda.memory.MemoryPointer(mem, 0)
 
     return ptr
+
+
+def register_reinitialize_hook(func, *args, **kwargs):
+    """
+    Add a function to the list of functions that will be called before
+    `rmm.reinitialize()`.
```

Comment:
Does this link correctly in the generated docs?

Reply:
Fix'd